Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/oryx/build:20200312.2
+FROM mcr.microsoft.com/oryx/build:github-actions-20200421.4
```

Comment:
Did you checkout this master branch to v1 branch? Before merging this PR in, can you push the v1 branch to server first

Reply:
Do you mean by updating v1 tag to current master branch?

Reply:
We already have a `v1` tag so we are good. Once you merge this PR, you can create a new tag called `V2` which also points to master